crates/solidity/testing/snapshots/bindings_output/expressions/type_expr/input.sol

```diff
@@ -1,7 +1,17 @@
+enum Answer {
```

Comment:
It would be preferable to have a separate test for this, and make it as small as possible. Another suggestion is to use a library for extending the type, as that approach is compatible with all versions of Solidity.